### ISSUE TITLE
Add option to merge groups by name instead of UUID

### ISSUE THREAD

**Issue by bitstreamout** [keepassxreboot/keepassxc#3957]
that is that the latest entry should win over the older entry.  This would allow to use kdbx databases over several hosts.

**Comment by droidmonkey** [comment]
I tend to agree with this behavior, but some people would find it bad behavior since technically two groups with different UUID's are different groups. It would be a nice option to have though for those that don't care about the technical correctness.

**Comment by droidmonkey** [comment]
We won't be implementing this
